common/common.tf

```diff
@@ -44,7 +34,17 @@ variable "dynamodb_table" {
   type        = string
 }
 
+variable "env" {
+  description = "Environment being deployed to"
+  type        = string
+}
+
 variable "key" {
   description = "Key that tfstate is stored in"
   type        = string
 }
+
+variable "region" {
+  description = "Region resources are being deployed to"
+  type        = string
+}
```

Comment:
The pre-commit-check pipeline is failing on this newline and it looks like you're using an old terraform version. Hold on one sec I'll push up my branch 
```
Run pre-commit run -a
[INFO] Initializing environment for git://github.com/antonbabenko/pre-commit-terraform.
[INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Installing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
Terraform fmt............................................................Passed
Terraform docs...........................................................Failed
- hook id: terraform_docs
- files were modified by this hook
Fix End of Files.........................................................Failed
- hook id: end-of-file-fixer
- exit code: 1
- files were modified by this hook

Fixing common/common.tf

Trim Trailing Whitespace.................................................Passed
Don't commit to branch...................................................Passed
##[error]Process completed with exit code 1.
```

Reply:
I sent your PR a PR https://github.com/cdaniluk/sample-aws-project/pull/1